release-calendar/webpack.client.config.js

```diff
@@ -59,5 +67,6 @@ module.exports = {
   externals: {
     'react': 'React',
     'react-dom': 'ReactDOM',
+    'react-native-sqlite-storage': 'react-native-sqlite-storage',
```

Comment:
Why was this added?

Reply:
was to limit large bundle size, other solution found